docs/Tutorials/Create-IBFT-Network.md

```diff
@@ -0,0 +1,309 @@
+```
+
+The command line specifies: 
+
+* Data directory for Node-2 using the [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) option.
+* Different port to Node-1 for P2P peer discovery using the [`--p2p-port`](../Reference/Pantheon-CLI-Syntax.md#p2p-port) option.
+* Different port to Node-1 for HTTP JSON-RPC using the [`--rpc-http-port`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-port) option.
+* Enode URL for Node-1 using the [`--bootnodes`](../Reference/Pantheon-CLI-Syntax.md#bootnodes) option.
+* Other options as for [Node-1](#5-start-first-node-as-bootnode).
+
+
+### 10. Start Node-3
+
+Start another terminal, change to the `Node-3` directory and start Node-3 replacing the enode URL with your bootnode: 
+
+```bash tab="MacOS"
+pantheon --data-path=Node-3-data-path --genesis-file=../ibftGenesis.json --bootnodes="enode://<node public key ex 0x>@127.0.0.1:30303" --p2p-port=30305 --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8547    
+```
+
+```bash tab="Windows"
+pantheon --data-path=Node-3-data-path --genesis-file=..\ibftGenesis.json --bootnodes="enode://<node public key ex 0x>@127.0.0.1:30303" --p2p-port=30305 --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8547    
+```
+
+The command line specifies: 
+
+ * Data directory for Node-3 using the [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) option.
+ * Different port to Node-1 and Node-2 for P2P peer discovery using the [`--p2p-port`](../Reference/Pantheon-CLI-Syntax.md#p2p-port) option.
+ * Different port to Node-1 and Node-2 for HTTP JSON-RPC using the [`--rpc-http-port`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-port) option.
+ * Bootnode as for [Node-2](#6-start-node-2).
+ * Other options as for [Node-1](#5-start-first-node-as-bootnode). 
+
+### 11. Confirm Private Network is Working 
+
+Start another terminal, use curl to call the JSON-RPC API [`net_peerCount`](../Reference/JSON-RPC-API-Methods.md#net_peercount) method and confirm the nodes are functioning as peers: 
+
+```bash
+curl -X POST --data '{"jsonrpc":"2.0","method":"net_peerCount","params":[],"id":1}' 127.0.0.1:8545
+```
+
+The result confirms Node-1 has two peers (Node-2 and Node-3):
+```json
+{
+  "jsonrpc" : "2.0",
+  "id" : 1,
+  "result" : "0x2"
+}
+```
+
+## Next Steps 
+
+Look at the logs displayed to confirm Node-1 is producing blocks and Node-2 and Node-3 are importing blocks. 
+
+Use the [IBFT API to add](../Consensus-Protocols/Clique.md#adding-and-removing-signers) Node-2 or Node-3 as a validator. 
+
+!!! note
+    To add Node-2 or Node-3 as a validator you need the [node address as when specifying Node-1](#2-get-address-for-node-1) as the initial validator. 
+
```

Comment:
I got a bit lost in the adding of validators but that isn't part of this PR

Reply:
I'll look at making this clearer after the release.